cypress/integration/pages/frontPage/tests.js

```diff
@@ -92,26 +92,50 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
           });
         });
 
-        if (serviceHasIndexAlsos(service)) {
-          it('should contain Index Alsos at a mobile view', () => {
-            cy.viewport('iphone-5');
-            cy.get('section li')
-              .eq(0)
-              .within(() => {
-                cy.get('div div div a')
+        it('should contain Index Alsos at a mobile and desktop view, if relatedItems block exists', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+            ({ body }) => {
+              const topstories = body.content.groups[0].items[0];
+              const relatedItemsExists = Object.prototype.hasOwnProperty.call(
+                topstories,
+                'relatedItems',
+              );
+
+              const runIndexAlsoTests = () => {
+                cy.get('[aria-labelledby="Top-stories"]')
                   .eq(0)
                   .within(() => {
-                    cy.get('span').then($el => {
-                      if ($el.length > 1) {
-                        cy.get('svg').should('be.visible');
-                      } else {
-                        expect($el).not.to.have.descendants('svg');
-                      }
-                    });
+                    cy.get('div')
+                      .eq(10)
+                      .within(() => {
+                        cy.get('h4')
+                          .eq(0)
+                          .then($el => {
+                            expect($el.text()).includes(
+                              `${appConfig[service].translations.relatedContent}`,
+                            );
+                          });
+
+                        if (topstories.relatedItems.length > 1) {
+                          cy.get('ul li a');
```

Comment:
Does `cy.get('ul li a');` assert anything?

Reply:
It check whether the link within that structure exist. If it can't find it, the test will error. I can add a `should('be.visible')` to make that more clear